### PR TITLE
Branding

### PR DESCRIPTION
**Pull request by ndiesslin**
Added some styles to match the Macchina aesthetics.

### CODE REVIEW

File: styles/website.scss [hunk 1]
```diff
@@ -0,0 +1,112 @@
+// variablesaj
```
Comment (adamvoss) [inline styles/website.scss]: does the `aj` mean something, or perhaps accidental characters?

**Comment by adamvoss** [comment]
Hmm... I can't use `gitbook serve` with this:

```
#
# Fatal error in ../deps/v8/src/api.cc, line 1200
# Check failed: !value_obj->IsJSReceiver() || value_obj->IsTemplateInfo().
#

==== C stack trace ===============================

    node(v8::base::debug::StackTrace::StackTrace()+0x16) [0x1586c26]
    node(V8_Fatal+0xd6) [0x1581976]
    node(v8::Template::Set(v8::Local<v8::Name>, v8::Local<v8::Data>, v8::PropertyAttribute)+0xf7) [0xa981b7]
    /home/user/dev/macchina-docs/node_modules/node-sass/vendor/linux-x64-57/binding.node(SassTypes::Boolean::get_constructor()+0x381) [0x7f76f0b05331]
    /home/user/dev/macchina-docs/node_modules/node-sass/vendor/linux-x64-57/binding.node(SassTypes::Factory::initExports(v8::Local<v8::Object>)+0x197) [0x7f76f0b08767]
    /home/user/dev/macchina-docs/node_modules/node-sass/vendor/linux-x64-57/binding.node(RegisterModule(v8::Local<v8::Object>)+0x7a0) [0x7f76f0afedf0]
    node() [0x1366f95]
    node(v8::internal::FunctionCallbackArguments::Call(void (*)(v8::FunctionCallbackInfo<v8::Value> const&))+0x193) [0xac5493]
    node() [0xb43f48]
    node(v8::internal::Builtin_HandleApiCall(int, v8::internal::Object**, v8::internal::Isolate*)+0xb9) [0xb44969]
    [0x1a6f52b040bd]
Illegal instruction
```

File: styles/website.scss (hunk 1 above)
Comment (ndiesslin) [inline styles/website.scss]: Oh gosh, I must have accidentally added the aj. I can update this later!

**Comment by adamvoss** [comment]
Looks sharp!  Thanks!

There is a rush on getting this out so I went ahead and pushed some changes to fix the build and let the Sass issue be addressed later.  I question all the links being orange, but will open a new issue for that.

**Comment by ndiesslin** [comment]
Thanks, Adam!

**Pull request merged**